OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m

```diff
@@ -160,6 +160,10 @@ - (void)viewWillAppear:(BOOL)animated {
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(willEnterForeground:) name:UIApplicationWillEnterForegroundNotification object:nil];
 
     [self reloadDataAnimated:NO];
+    
+    OBATripDeepLink *deepLink = [[OBATripDeepLink alloc] initWithArrivalAndDeparture:_arrivalAndDeparture region:self.modelDAO.currentRegion];
```

Comment:
please use property syntax (i.e. `self.arrivalAndDeparture`) everywhere but in `-init` and `-dealloc`.